ext/standard/var_unserializer.c

```diff
@@ -164,6 +260,7 @@ PHPAPI void var_destroy(php_unserialize_data_t *var_hashx)
 		efree_size(var_dtor_hash, sizeof(var_dtor_entries));
 		var_dtor_hash = next;
 	}
+	var_wakeup_all(var_hashx, 1); /* Free the copies of objects (which would call __wakeup) if we haven't already. */
```

Comment:
I think it would be clearer to write this the same way as for var_dtor_hash -- I found the intermixed wakeup/no-wakeup code in var_wakeup_all pretty hard to follow.